tsconfig.json

```diff
@@ -40,5 +40,24 @@
   "include": [
     "./build/",
     "./recipes/"
-  ]
+  ],
+  "exclude": [
+    "node_modules",
+    "**/build/**",
+    "**/fixtures/**",
+    "**/test/**",
+    "**/*.fixture.mjs",
+    "**/*.fixture.mts",
+    "**/*.fixture.js",
+    "**/*.fixture.ts",
+    "**/*.mock.mjs",
+    "**/*.e2e.mjs",
+    "**/*.e2e.mts",
+    "**/*.e2e.js",
+    "**/*.e2e.ts",
+    "**/*.spec.mjs",
+    "**/*.spec.mts",
+    "**/*.spec.js",
+    "**/*.spec.ts",
```

Comment:
This is super verbose, and the tsconfig docs specifically say they support full globs, but using `{mjs,mts,js,ts}` silently fails.